app/controllers/v1/base_controller.rb

```diff
@@ -11,17 +14,30 @@ def rescue_error_messages
     end
   end
 
-  rescue_from ActionController::ParameterMissing do |exception|
-    render json: { error: "#{exception.param} is required" }, status: 422
+  def missing_parameter(error)
+    render json: { error: "#{error.param} is required" }, status: 422
+  end
+
+  def record_not_found(error)
+    render json: {error: error.message}, status: :not_found
+  end
+
+  def default_person_params
+    params.require(:person).permit(Person::ALL_AVAILABLE_ATTRIBUTES)
+  end
+
+  def person_from_params
```

Comment:
I agree for the params fields.  I'm sorta assuming we'll pull these peron params out when we have our constructor setup.  For the error handling, I'm not sure there is a better place.